File: repaso objetos/python/primer_objeto/primer_objeto.py
class Persona():
    def __init__(self, nombre, apellido):
        self.__nombre__ = nombre
        self.__apellido__ = apellido

    def get_nombre(self):
        return self.__nombre__

    def get_apellido(self):
        return self.__apellido__

class Saludador():
    def __init__(self, persona):
        self.__persona__ = persona
    def saludar(self,msg):
         return msg + self.__persona__.get_nombre()+" "+self.__persona__.get_apellido()
         
         
p  = Persona("Sebastián", "González")
s = Saludador(p)
print(s.saludar("Buenos días "))

File: repaso objetos/java/PrimerObjeto/PrimerObjeto.java
package objetos.java;
class Persona{
    private String nombre;
    private String apellido;

    public void setNombre(String nombre){
        this.nombre=nombre;
    }
    public void setApellido(String apellido){
        this.apellido=apellido;
    }
    public String getNombre(){
        return this.nombre;
    }
    public String getApellido(){
        return this.apellido;
    }
}
public class PrimerObjeto {
    public static void main(String[] args){
        Persona p= new Persona();
    
        p.setNombre ("Sebastian");
        p.setApellido("Gonzalez");

        Saludador s = new Saludador();
        s.setPersona(p);
        System.out.println(s.saludar("Hola "));
    }

}
class Saludador {
    private Persona p;

    public void setPersona(Persona p){
        this.p = p;
    }

    public String saludar(String msg){
        return msg + p.getNombre()+ " " + p.getApellido();
    }
}
